torch/_inductor/wrapper_benchmark.py

```diff
@@ -332,7 +356,7 @@ def ncu_analyzer(benchmark_name, benchmark_compiled_module_fn):
         return
 
 
-def collect_memory_snapshot(benchmark_compiled_module_fn):
+def collect_memory_snapshot(benchmark_compiled_module_fn: Callable[..., float]) -> None:
```

Comment:
```suggestion
def collect_memory_snapshot(benchmark_compiled_module_fn: Callable[P, float]) -> None:
```
Use ParamSpec to not erase the args to the Callable